src/benchUtil.c

```diff
@@ -616,4 +616,70 @@ void cleanup_taos_list() {
         taos_close(g_arguments->pool->taos_list[i]);
     }
     tmfree(g_arguments->pool->taos_list);
-}
\ No newline at end of file
+}
+void delay_list_init(delayList *list) {
+    list->size = 0;
+    list->head = NULL;
+    list->tail = NULL;
+}
+
+void delay_list_destroy(delayList *list) {
+    while (list->size > 0) {
+        delayNode *current = list->head;
+        list->head = list->head->next;
```

Comment:
is it possible if list->head is NULL?

Reply:
I think if list->size is large than 0, then list->head will not be NULL, it will not go into this loop